zanata-war/src/main/webapp/WEB-INF/pages.xml

```diff
@@ -24,15 +24,10 @@
   </page>
 
 
-  <page view-id="/dashboard/*" login-required="true">
-    <restrict>#{identity.loggedIn}</restrict>
-  </page>
-
   <page view-id="/edit_home_content.xhtml">
     <action execute="#{breadcrumbs.clear}" />
     <action
       execute="#{breadcrumbs.addLocation('', msgs['jsf.server.EditHomePage.label'])}" />
-    <restrict>#{s:hasRole('admin')}</restrict>
```

Comment:
The page itself is secured by jsf in webapp/home.xhtml:19. Basically hidden from user without admin role. 

----- Original Message -----

> From: "Sean Flanigan" <notifications@github.com>
> To: "zanata/zanata-server" <zanata-server@noreply.github.com>
> Cc: "Patrick Huang" <pahuang@redhat.com>
> Sent: Friday, July 31, 2015 11:30:21 AM
> Subject: Re: [zanata-server] Replace Seam Security (#908)

> In zanata-war/src/main/webapp/WEB-INF/pages.xml :
> >    <page view-id="/edit_home_content.xhtml">
> >      <action execute="#{breadcrumbs.clear}" />
> >      <action
> >        execute="#{breadcrumbs.addLocation('',
> >        msgs['jsf.server.EditHomePage.label'])}" />
> > -    <restrict>#{s:hasRole('admin')}</restrict>

> Where did this check move to? I can see @CheckRole("admin") on
> ServerConfigurationBean; is there anything else?

> —
> Reply to this email directly or view it on GitHub .